gobblin-compaction/src/main/java/gobblin/compaction/CompactionRunner.java

```diff
@@ -75,7 +83,9 @@ public static void main(String[] args) throws ConfigurationException, IOExceptio
       }
     }
     LOG.info("Found " + numOfJobs + " compaction tasks.");
-    PrintWriter pw = new PrintWriter(properties.getProperty(TIMING_FILE, TIMING_FILE_DEFAULT));
+    PrintWriter pw = new PrintWriter(
+        new OutputStreamWriter(new FileOutputStream(properties.getProperty(TIMING_FILE, TIMING_FILE_DEFAULT)),
+            Charset.defaultCharset()));
```

Comment:
The default character set depends on the locale setting where the JVM is running. This is dangerous if the locale suddenly changes on we read/write the same data on a different machine with a different locale.

The best thing is to use a fixed charset -- UTF-8.